eo-runtime/src/main/java/org/eolang/ExprReduce.java

```diff
@@ -123,20 +117,92 @@ public Phi get(final Phi rho) {
             final Object val = new Dataized(args[idx]).take();
             if (!val.getClass().getCanonicalName().equals(this.type.getCanonicalName())) {
                 throw new ExFailure(
-                    "The %dth argument of '%s' is not a(n) %s: %s",
-                    idx + 1, this.oper, this.type.getSimpleName(), val
+                "The %dth argument of '%s' is not a(n) %s: %s",
+                idx + 1, this.oper, this.type.getSimpleName(), val
                 );
             }
             final T typed = this.type.cast(val);
             final String msg = this.validation.apply(typed);
             if (!msg.isEmpty()) {
                 throw new ExFailure(
-                    "The %dth argument of '%s' is invalid: %s",
-                    idx + 1, this.oper, msg
+                "The %dth argument of '%s' is invalid: %s",
+                idx + 1, this.oper, msg
                 );
             }
             acc = this.reduction.apply(acc, typed);
         }
         return new Data.ToPhi(acc);
     }
+
+    /**
+     * Builds a class that stores some attributes of ExprReduce object.
+     * <p/>Definition example:
+     * <code><pre>
+     *     final VarargExpr&lt;Long&gt; st = new Storage<>(
+     *         "plus",
+     *         "x",
+     *         Long.class
+     *     );
+     * </pre></code>
+     * @param <T> Type of arguments
+     * @since 1.0
+     */
+    public static final class Storage<T> {
```

Comment:
`<T>` Type parameter for this class is quite misleading: what arguments are assumed?

Reply:
@mximp Ok, will change